Editor/Modules/TextureAnalyzer.cs

```diff
@@ -158,7 +155,7 @@ public IEnumerable<ProjectIssue> Analyze(ProjectAuditorParams projectAuditorPara
                     .WithLocation(textureImporter.assetPath);
             }
 
-            if (!textureImporter.streamingMipmaps && texture.width > TEXTURE_WIDTH && texture.height > TEXTURE_HEIGHT)
+            if (!textureImporter.streamingMipmaps && size > projectAuditorParams.settings.TextureStreamingMipmapsMinimumSize)
```

Comment:
I think this should be:

size > projectAuditorParams.settings.TextureStreamingMipmapsMinimumSize * projectAuditorParams.settings.TextureStreamingMipmapsMinimumSize

...so we check the total texture (pixel) size against the _square_ of for example 1000.